newm-server/src/main/kotlin/io/newm/server/features/distribution/model/AddUserResponse.kt

```diff
@@ -0,0 +1,17 @@
+package io.newm.server.features.distribution.model
+
+import io.newm.shared.serialization.UUIDSerializer
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import java.util.UUID
+
+@Serializable
+data class AddUserResponse(
+    @SerialName("message")
+    val message: String,
+    @SerialName("success")
+    val success: Boolean,
+    @SerialName("uuid")
+    @Serializable(with = UUIDSerializer::class)
+    val uuid: UUID,
+)
```

Comment:
Any possibility of a missing filed on this response? If so, maybe make fields nullable.